content/en/docs/reference/labels-annotations-taints/_index.md

```diff
@@ -1033,6 +1033,21 @@ and legacy in-tree cloud providers), it sets this annotation on the Node to deno
 set from the command line flag (`--node-ip`). This IP is verified with the cloud provider as valid
 by the cloud-controller-manager.
 
+### volume.alpha.kubernetes.io/node-affinity (alpha) {#volume-alpha-kubernetes-io-node-affinity}
+
+Type: Annotation
+
+Example: `volume.alpha.kubernetes.io/node-affinity: "{}"`
+
+Used on: PersistentVolume
+
+The `volume.alpha.kubernetes.io/node-affinity` annotation was used to specify 
+experimental node affinity rules for PersistentVolumes (PV) in Kubernetes.
+Volume _[node affinity](/docs/concepts/storage/persistent-volumes/#node-affinity)_
+allows you to restrict the scheduling of Pods that use the corresponding PersistentVolume
+so that the Pods can only bound to appropriate nodes in your cluster, based on
```

Comment:
The logic is confusing ...
I'm supposed to set this on a PV to impact the scheduling of a Pod?
Sounds that implies an annotation on the Pod rather than the PV, right?
I'm not familiar with the background, so forgive me about the dumb questions.